src/react-apps/applications/receipt/__tests__/App.test.tsx

```diff
@@ -0,0 +1,6 @@
+// Just a hard-coded test to test that jest runs
```

Comment:
I don't know if this test (of Jest) is necesarry.

Reply:
It's in the boilerplate for new applications. It's not needed, but is a good way to test that jest runs correctly in the initialization of new apps.

Reply:
Removed.